src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/TravellerSelection.tsx

```diff
@@ -111,17 +115,18 @@ export function TravellerSelection({
           .map((u) => `${u.count} ${getReferenceDataName(u, language)}`)
           .join(', ');
 
-  const newLabelAccessibility = isOnBehalfOfEnabled
-    ? screenReaderPause + t(LabelInfoTexts.labels['new'])
-    : '';
+  const newLabelAccessibility =
+    isOnBehalfOfEnabled && canSelectUserProfile
+      ? screenReaderPause + t(LabelInfoTexts.labels['new'])
+      : '';
 
   const sendingToOthersAccessibility = isOnBehalfOfToggle
     ? screenReaderPause + t(PurchaseOverviewTexts.onBehalfOf.sendToOthersText)
     : '';
 
   const accessibility: AccessibilityProps = {
     accessible: true,
-    accessibilityRole: 'button',
+    accessibilityRole: canSelectUserProfile ? 'button' : 'none',
     accessibilityLabel:
```

Comment:
Do a quick check with UX regarding if "Selected travellers:" still is the preferred label for this, when the user can't actually select.

Reply:
I'll consult with @ckbilstad in a few minutes

Reply:
Updated to "Traveller:" as discussed with @ckbilstad